app/src/app/uploader/page.tsx

```diff
@@ -0,0 +1,154 @@
+'use client';
+import React, {useState} from 'react';
+import Papa from 'papaparse';
+import {ErrorNotification} from '../components/ErrorNotification';
+import {GerryDBViewSelector} from '../components/sidebar/GerryDBViewSelector';
+import {useMapStore} from '@/app/store/mapStore';
+import {Assignment, createMapDocument, uploadAssignments} from '@/app/utils/api/apiHandlers';
+
+type MapLink = {
+  document_id: string;
+  name: string;
+};
+
+export default function Uploader() {
+  const [progress, setProgress] = useState<number>(0);
```

Comment:
Q: The progress bar doesn't seem to work in the current implementation, although the response time to process the query isn't very long. Do we need a progress bar?

Reply:
I'll remove progress bar